ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -625,20 +625,20 @@ type IterRef blk m r = Reference (Opaque (TestIterator m blk)) r
 -- | Mapping between iterator references and mocked iterators
 type KnownIters blk m r = RefEnv (Opaque (TestIterator m blk)) IteratorId r
 
--- | Concrete or symbolic references to a real reader
-type ReaderRef blk m r = Reference (Opaque (TestReader m blk)) r
+-- | Concrete or symbolic references to a real follower
+type FollowerRef blk m f = Reference (Opaque (TestFollower m blk)) f
```

Comment:
I think this `r` stood for "reference", not for "reader". EG This module is using quickcheck-state-machine, and the README in QSM's repo uses `r` in that same way as an abstraction over `IORef`. https://github.com/advancedtelematic/quickcheck-state-machine#readme